application/dash_application/views/call_ready_rna.py

```diff
@@ -201,7 +203,7 @@ def layout(query_string):
                                           {"display": "inline-block"},
                                           sidebar_utils.construct_jira_link([], title))]),
             html.Div(className="row flex-container", children=[
-                html.Div(className="sidebar four columns", children=[
+                html.Div(className="sidebar three columns", children=[
```

Comment:
Some interesting things happen to the sidebar when the browser window is at 50% width 
![image](https://user-images.githubusercontent.com/17747491/75170647-697e8980-56f8-11ea-8372-287bb4b6aa13.png)

Reply:
Well that's impressively ugly. I'll update it to four + eight columns which will still give us more space for graphs.

Reply:
Updated to use `one-third column` and `two-thirds column`, please let me know how that looks.

(It turns out using `four columns` and `eight columns` looks terrible because there are two sets of `@media (min-width: 550px)` queries, and the later (overriding) one only goes up to `seven columns` so it uses the `four columns` from the later media query and `eight columns` from the earlier query and they calculate element width differently so the combined width was too large for the screen and the tabs overflowed below the sidebar :((((((  )

Reply:
Seems better!
![image](https://user-images.githubusercontent.com/17747491/75360003-2ac90a80-5883-11ea-9796-5f22b87b5d58.png)
(The 'Approve this Run in MISO' button still gets cut off awkwardly, but idk if there's any helping that one)